beesdoo_easy_my_coop/views/subscription_templates.xml

```diff
@@ -4,35 +4,30 @@
 <odoo>
 
     <template
-        id="beesdoo_easy_my_coop.becomecooperator"
+        id="beesdoo_easy_my_coop.rules_template"
         name="Become Cooperator (in beesdoo_easy_my_coop)"
-        inherit_id="easy_my_coop_website.becomecooperator"
+        inherit_id="easy_my_coop_website.rules_template"
     >
         <xpath expr="//div[@id='internal_rules_approved']" position="before">
             <div
                 id="info_session_confirmed"
                 t-if="display_info_session"
                 t-attf-class="form-group"
             >
-                <label
-                    class="col-md-3 col-sm-4 control-label"
-                    for="info_session_confirmed"
-                >
+                <label for="info_session_confirmed">
                     Info Session
                 </label>
-                <div class="col-md-9 col-sm-8">
                     <div class="form-check">
                         <input
-                            type="checkbox"
-                            name="info_session_confirmed"
-                            class="form-check-input"
-                            t-att-required="info_session_required"
-                            t-att-checked="info_session_confirmed"
-                        />
-                        <div class="form-check-label">
+                        type="checkbox"
+                        name="info_session_confirmed"
+                        class="form-check-input"
+                        t-att-required="info_session_required"
+                        t-att-checked="info_session_confirmed"
+                    />
+                    <label>
                             <t t-raw="info_session_text" />
-                        </div>
-                    </div>
+                    </label>
```

Comment:
Bad indentation

Reply:
I don't think you successfully fixed it :x

Reply:
ah ignore my comment

Reply:
Fixed (I hope :smile: ), I used a reformat tool